pkg/service/presentation/model/model.go

```diff
@@ -78,6 +78,10 @@ type ListSubmissionResponse struct {
 	Submissions []Submission `json:"submissions"`
 }
 
+type ListDefinitionsResponse struct {
+	Definitions []*exchange.PresentationDefinition `json:"definitions"`
```

Comment:
is there a need for this to be a pointer?

Reply:
No need, but seemed unnecessary to copy the full struct.